src/mlpack/core/data/one_hot_encoding.hpp

```diff
@@ -0,0 +1,43 @@
+/**
+ * @file one_hot_encoding.hpp
+ * @author Jeffin Sam
+ *
+ * One hot encoding functions
+ * The purpose of this function is to convert
+ * categorical variables as binary vectors.
```

Comment:
Looks like these lines are shorter than 80 characters.  Maybe you can combine them?

Reply:
Wrapped up